ytmusicapi/mixins/browsing.py

```diff
@@ -8,15 +8,17 @@
 
 
 class BrowsingMixin:
-    def search(self, query: str, filter: str = None) -> List[Dict]:
+    def search(self, query: str, filter: str = None, maxResults: int = 20) -> List[Dict]:
```

Comment:
This parameter should be named limit to be consistent with other function calls

Reply:
Will do.